index.js

```diff
@@ -1,12 +1,15 @@
-export { SpeechRecognition } from './js/src/SpeechRecognition';
+import { SpeechRecognition } from './js/src/SpeechRecognition';
+
+export * from './js/src/SpeechRecognition';
+
 if (Promise === undefined) {
     throw new Error('Promises are not supported in this browser. Use this polyfill: https://www.npmjs.com/package/es6-promise');
 }
 // polyfill mediaDevices.getUserMedia
 if (!navigator.mediaDevices) navigator.mediaDevices = {};
 navigator.mediaDevices.getUserMedia = navigator.mediaDevices.getUserMedia || (function () {
     // returns a getUserMedia function
-    var getUserMedia = navigator.getUserMedia || navigator.webkitGetUserMedia || navigator.mozGetUserMedia;
+    let getUserMedia = navigator.getUserMedia || navigator.webkitGetUserMedia || navigator.mozGetUserMedia;
```

Comment:
this file is not included in the transpilation process. A polyfill should support older browsers and not all support `let` for variable declarations.